src/main/java/org/molgenis/vcf/report/model/Person.java

```diff
@@ -0,0 +1,34 @@
+package org.molgenis.vcf.report.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.experimental.NonFinal;
+
+@Value
+@NonFinal
+public class Person {
+  @JsonProperty("familyId")
+  @NonNull
+  String familyId;
+
+  @JsonProperty("individualId")
+  @NonNull
+  String individualId;
+
+  @JsonProperty("maternalId")
+  @NonNull
+  String maternalId;
```

Comment:
request: could we switch the order of maternalId and paternalId (which will change the order in the constructor!) to align with the order of https://phenopackets-schema.readthedocs.io/en/latest/pedigree.html#rstpedigree?